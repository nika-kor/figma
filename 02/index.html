<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>MoGo</title>
    <link rel="stylesheet" href="style.css">
</head>
<body>
 <div class="container">
    <header>
        <nav>
             <ul style="list-style:none; columns: 6;">
               <li><a href="" style="text-decoration: none; color: #95E1D3; font-size: 30px;">MoGo</a></li>
               <li><a href="" style="text-decoration: none; color: #95E1D3; font-size: 14px;">ABOUT</a></li>
               <li><a href="" style="text-decoration: none; color: #95E1D3; font-size: 14px;">SERVICE</a></li>
               <li><a href="" style="text-decoration: none; color: #95E1D3; font-size: 14px;">WORK</a></li>
               <li><a href="" style="text-decoration: none; color: #95E1D3; font-size: 14px;">BLOG</a></li>
               <li><a href="" style="text-decoration: none; color: #95E1D3; font-size: 14px;">CONTACT</a></li>
             </ul>
        </nav>
    </header>

    <main>
        <h3 class="title">We work with</h3>
        <h2>Amazing Services</h2>
       <hr class="line" >
        <ul style="list-style:none; columns: 3;" class="list">
            <li>
                <h4>photography</h4>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
            </li>
            <li>
                <h4>photography</h4>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
            </li>
            <li>
                <h4>photography</h4>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
            </li>
            <li>
                <h4>photography</h4>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
            </li>
            <li>
                <h4>photography</h4>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
            </li>
            <li>
                <h4>photography</h4>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
            </li>
        </ul>
        <section>
            <h3 class="title">Who we are</h3>
            <h2>Meet our team</h2>
            <hr class="line">
            <p class="title" style="font-size: 15px; color:#B6B6B6;">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
        </section>
            <ul style="list-style:none; columns: 3;">
                <li class="card">
                    <img src="./image/first-man 1.png" alt="first-man">
                    <h5 style="font-size: 14px; color:#333333; text-align: center;">Matthew Dix</h5>
                    <p style="font-size: 15px; color:#B6B6B6; text-align: center;">Graphic Design</p>
                </li class="card">
                <li>
                    <img src="./image/second-man 1.png" alt="second-man">
                    <h5 style="font-size: 14px; color:#333333; text-align: center;">Christopher Campbell</h5>
                    <p style="font-size: 15px; color:#B6B6B6; text-align: center;">Branding/UX design</p>
                </li>
                <li class="card">
                    <img src="./image/third-man 1.png" alt="third-man">
                    <h5 style="font-size: 14px; color:#333333; text-align: center;">Michael Fertig </h5>
                    <p style="font-size: 15px; color:#B6B6B6; text-align: center;">Developer</p>
                </li>
            </ul>
        </ul>
    </main>
    <hr class="line2">
    <footer>
        <p class="copyright">Â© 2016 MoGo free PSD template by <span style="color: #F38181;">Laaqiq</span></p>
        <form>
            <label><input type="email" name="email" placeholder="Your Email..." /></label>
            <button>Subscribe</button>
        </form>
    </footer>
</div>
</body>
</html>