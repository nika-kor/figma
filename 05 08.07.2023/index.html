<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>

    <header>
        <nav>
            <ul style="list-style:none; columns: 4;">
                <li><a href="">LIFESTYLE</a></li>
                <li><a href="">PHOTODIARY</a></li>
                <li><a href="">MUSIC</a></li>
                <li><a href="">TRAVEL</a></li>
            </ul>
        </nav>
    </header>

    <main>
        <ul style="list-style:none; columns: 2;">
            <li>
                <img src="./image/1 1.png" alt="ok">
                <p>LIFESTYLE</p>
                <h2>More than just a music festival</h2>
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea.</p>
            </li>

            <li>
                <img src="./image/2 1.png" alt="ok">
                <p>LIFESTYLE</p>
                <h2>Life tastes better with coffee</h2>
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea.</p>
            </li>

            <li>
                <img src="./image/3 1.png" alt="ok">
                <p>PHOTODAIRY</p>
                <h2>American dream</h2>
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea.</p>
            </li>

            <li>
                <img src="./image/4 1.png" alt="ok">
                <p>PHOTODAIRY</p>
                <h2>A day exploring the Alps</h2>
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea.</p>
            </li>

            <li>
                <img src="./image/1 2.png" alt="ok">
                <p>LIFESTYLE</p>
                <h2>Top 10 song for running</h2>
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea.</p>
            </li>

            <li>
                <img src="./image/2 2.png" alt="ok">
                <p>LIFESTYLE</p>
                <h2>Cold winter days</h2>
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea.</p>
            </li>
        </ul>

        <button>Load more</button>
        
    </main>
</body>
</html>