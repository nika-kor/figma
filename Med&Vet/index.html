<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Med&Vet</title>
    <link rel="stylesheet" href="reset.css">
    <link rel="stylesheet" href="style.css">
</head>
<body>
    <header>
        <div class="wrap nav-wrap">
                <a href="./index.html" class="logo">Med&Vet <img src="./image/logo.png" alt="logo"></a>
    
                <nav class="nav">
                    <a href="#service" class="nav-link">Services</a>
                    <a href="" class="nav-link">Doctors</a>
                    <a href="" class="nav-link">Benefits</a>
                    <a href="" class="nav-link">Reviwes</a>
                    <a href="" class="nav-link">Contacts</a>
                </nav>
        </div>
    </header>

    <main>

        <!-- hero -->

        <section class="hero">
            <div class="wrap wrap-hero">
                <div>
                    <p class="hero-desc-first">Number 1 in Canada</p>
                    <h1 class="hero-title">The best veterinary clinic for your pets</h1>
                    <p class="hero-desc-second">With us your pets would be safe-and-sound</p>
                    <button class="hero-btn">Book an apointment</button>
                </div>
                <div class="image-hero"><img src="./image/dogmain.png" alt=""></div>
            </div>
        </section>

        <!-- Our Services -->

        <section class="service" id="service">
            <div class="wrap">
                <h2 class="service-title"></h2>

                <ul class="service-list">
                    <li class="service-list-item">
                        <div class="service-item-wrap-first">
                            <img src="./image/dog.png" alt="">
                        </div>
                        <div class="service-item-wrap-second">
                            <h5 class="service-item-title">Wellness <span class="service-item-title-span">Care</span></h5>
                            <p class="service-item-desc">Our wellness programs include: comprehensive physical exam; internal and external parasite testing.</p>
                        </div>
                    </li>
                    <li class="service-list-item">
                        <div class="service-item-wrap-first">
                            <img src="./image/hart.png" alt="">
                        </div>
                        <div class="service-item-wrap-second">
                            <h5 class="service-item-title">Anesthetic <span class="service-item-title-span">Monitoring</span></h5>
                            <p class="service-item-desc">Your pet will be examined during check in to ensure that he or she is in top shape for surgery that day.</p>
                        </div>
                    </li>
                    <li class="service-list-item">
                        <div class="service-item-wrap-first">
                            <img src="./image/bowl.png" alt="">
                        </div>
                        <div class="service-item-wrap-second">
                            <h5 class="service-item-title">Nutritional <span class="service-item-title-span">Counseling</span></h5>
                            <p class="service-item-desc">Some pets require special prescription food and all pets benefit from a balanced diet.</p>
                        </div>
                    </li>
                    <li class="service-list-item">
                        <div class="service-item-wrap-first">
                            <img src="./image/stethoscope.png" alt="">
                        </div>
                        <div class="service-item-wrap-second">
                            <h5 class="service-item-title">Pain <span class="service-item-title-span">Management</span></h5>
                            <p class="service-item-desc">We understand that our patients feel pain and discomfort under the same circumstances as people do.</p>
                        </div>
                    </li>
                    <li class="service-list-item">
                        <div class="service-item-wrap-first">
                            <img src="./image/syringe.png" alt="">
                        </div>
                        <div class="service-item-wrap-second">
                            <h5 class="service-item-title">Vaccination <span class="service-item-title-span">Care</span></h5>
                            <p class="service-item-desc">When a baby kitten or puppy is born, its immune system is not yet mature; the baby is wide open for infection.</p>
                        </div>
                    </li>
                    <li class="service-list-item">
                        <div class="service-item-wrap-first">
                            <img src="./image/dental.png" alt="">
                        </div>
                        <div class="service-item-wrap-second">
                            <h5 class="service-item-title">Dental <span class="service-item-title-span">Care</span></h5>
                            <p class="service-item-desc">Regular dental cleanings are important in maintaining not only your pet’s teeth but his or hev overall health as well.</p>
                        </div>
                    </li>
                </ul>
            </div>
        </section>

        <!-- How we work? -->

        <section class="work">
            <div class="wrap">
                <h2 class="work-title">How we work?</h2>
                <p class="work-desc">Our system is huge, some clients don't immediately understand what needs to be done, so we’ve created this video to avoid misunderstandings</p>
            </div>
        </section>

        <!-- Our Specialists -->

        <section class="specialist">
            <div class="wrap">
                <h2 class="specialist-title">Our Specialists</h2>
                <ul class="specialist-list">

                    <li class="specialist-list-item">
                        <img src="./image/humanfirst.png" alt="Jonathan Kanes" class="specialist-img">
                        <h5 class="specialist-name">Jonathan Kanes</h5>
                    </li>

                    <li class="specialist-list-item">
                        <img src="./image/humansecond.png" alt="Mike Goodwill" class="specialist-img">
                        <h5 class="specialist-name">Mike Goodwill</h5>
                    </li>

                    <li class="specialist-list-item">
                        <img src="./image/humanthird.png" alt="Mervin Terance" class="specialist-img">
                        <h5 class="specialist-name">Mervin Terance</h5>
                    </li>

                </ul>
            </div>
        </section>
    </main>

    <footer>
        <div class="wrap">
            <div class="title-footer-wrap">
                    <a href="./index.html" class="logo"><img src="./image/logo.png" alt="logo"> Med&Vet</a>
            </div>
            <div class="footer-desc-wrap">
                <h3 class="footer-title">CONTACT</h3>
                <ul class="footer-list">
                    <li class="footer-list-item">
                        <a href="" class="footer-link">
                            <p class="footer-desc">24 Broadcast Drive Charlotte NC 28479, Canada </p>
                        </a>
                    </li>
                    <li class="footer-list-item">
                        <a href="" class="footer-link">
                            <p class="footer-desc">med&vet@mail.com</p>
                        </a>
                    </li>
                    <li class="footer-list-item">
                        <a href="" class="footer-link">
                            <p class="footer-desc">(528) 289 - 4790</p>
                        </a>
                    </li>
                </ul>
            </div>
        </div>
    </footer>
</body>
</html>