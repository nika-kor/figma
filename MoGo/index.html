<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>MoGo</title>
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Kaushan+Script&family=Montserrat:wght@400;700&family=Roboto:wght@300;400&display=swap" rel="stylesheet">

    <link rel="stylesheet" href="reset.css">
    <link rel="stylesheet" href="style.css">
</head>
<body>
    <header>
        <div class="wrap">
            <nav>
                <ul class="nav-list">
                <li class="nav-list-item"><a href="#" class="item-a-title">MoGo</a></li>
                <li class="nav-list-item"><a href="#" class="item-a">ABOUT</a></li>
                <li class="nav-list-item"><a href="#" class="item-a">SERVICE</a></li>
                <li class="nav-list-item"><a href="#" class="item-a">WORK</a></li>
                <li class="nav-list-item"><a href="#" class="item-a">BLOG</a></li>
                <li class="nav-list-item"><a href="#" class="item-a">CONTACT</a></li>
                </ul>
            </nav>
        </div>
    </header>

    <main>
        <div class="wrap">
            <h3 class="main-title">We work with</h3>
            <h2 class="title">Amazing Services</h2>
        <hr class="line" >
            <ul class="desc-list">
                <img src="./image/alarm 1.png" alt="" class="img-desc">
                <li class="desc-list-item">
                    <h4 class="item-title">photography</h4>
                    <p class="item-desc">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
                </li>
                <img src="./image/bar-graph 1.png" alt="" class="img-desc">
                <li class="desc-list-item">
                    <h4 class="item-title">photography</h4>
                    <p class="item-desc">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
                </li>
                <img src="./image/display 1.png" alt="" class="img-desc">
                <li class="desc-list-item">
                    <h4 class="item-title">photography</h4>
                    <p class="item-desc">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
                </li>
                <img src="./image/paper 1.png" alt="" class="img-desc">
                <li class="desc-list-item">
                    <h4 class="item-title">photography</h4>
                    <p class="item-desc">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
                </li>
                <img src="./image/home 1.png" alt="" class="img-desc">
                <li class="desc-list-item">
                    <h4 class="item-title">photography</h4>
                    <p class="item-desc">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
                </li>
                <img src="./image/digital-marketing 1.png" alt="" class="img-desc">
                <li class="desc-list-item">
                    <h4 class="item-title">photography</h4>
                    <p class="item-desc">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
                </li>
            </ul>
            <section>
                <h3 class="main-title">Who we are</h3>
                <h2  class="title">Meet our team</h2>
                <hr class="line">
                <p class="title-desc">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
            </section>
                <ul class="comand-list">
                    <li class="comand-list-item">
                        <img src="./image/first-man 1.png" alt="first-man">
                        <h5 class="comand-item-title">Matthew Dix</h5>
                        <p class="comand-item-desc">Graphic Design</p>
                    </li>
                    <li class="comand-list-item">
                        <img src="./image/second-man 1.png" alt="second-man">
                        <h5 class="comand-item-title">Christopher Campbell</h5>
                        <p class="comand-item-desc">Branding/UX design</p>
                    </li>
                    <li class="comand-list-item">
                        <img src="./image/third-man 1.png" alt="third-man">
                        <h5 class="comand-item-title">Michael Fertig </h5>
                        <p class="comand-item-desc">Developer</p>
                    </li>
                </ul>
            </ul>
        </div>
    </main>
    <footer>
            <hr class="line2">
        <div class="wrap-footer">
            <p class="copyright">Â© 2016 MoGo free PSD template by <span class="copyright-span">Laaqiq</span></p>
            <form>
                <label><input type="email" name="email" placeholder="Your Email..." /></label>
                <button>Subscribe</button>
            </form>
        </div>
    </footer>
</body>
</html>