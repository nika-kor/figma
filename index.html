<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>
    <ul>
        <li><a href="./antools/index.html">antools</a></li>
        <li><a href="./Golden/index.html">Golden</a></li>
        <li><a href="./MoGo/index.html">MoGo</a></li>
        <li><a href="./photo/index.html">photo</a></li>
    </ul>
</body>
</html>