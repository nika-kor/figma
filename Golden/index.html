<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Golden</title>
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Bilbo+Swash+Caps&family=Montserrat:wght@400;700&family=Roboto&display=swap" rel="stylesheet">

    <link rel="stylesheet" href="reset.css">
    <link rel="stylesheet" href="style.css">
</head>
<body>

    <!-- header -->

    <header>
        <div class="wrap">
            <nav class="nav">
                <a href="#" class="nav-title">Golden</a>
                <ul class="navs-list">
                    <li><a href="#" class="nav-desc">Home</a></li>
                    <li><a href="#" class="nav-desc">Services</a></li>
                    <li><a href="#" class="nav-desc">Portfolio</a></li>
                    <li><a href="#" class="nav-desc">About</a></li>
                    <li><a href="#" class="nav-desc">Contact</a></li>
                </ul>
            </nav>
        </div>
    </header>

    <main>
        <div class="wrap">
            <section class="main">
                <h2 class="main-title">Services</h2>
                <ul class="main-list">
                    <li class="main-list-item">
                        <img src="./image/134.png" alt="ok" class="main-img"></img>
                        <h3 class="main-item-title">E-Commerce</h3>
                        <p class="main-item-desc">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
                    </li>
                    <li class="main-list-item">
                        <img src="./image/132.png" alt="ok" class="main-img">
                        <h3 class="main-item-title">Responsive Web</h3>
                        <p class="main-item-desc">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
                    </li>
                    <li class="main-list-item">
                        <img src="./image/130.png" alt="ok" class="main-img">
                        <h3 class="main-item-title">Web Security</h3>
                        <p class="main-item-desc">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
                    </li>
                </ul>
            </section>
        </div>
    </main>

    <footer>
        <div class="wrap">
            <h2 class="footer-title">Contact us</h2>
            <form>
                    <ul class="footer-list">
                        <li class="footer-list-item3">
                            <input class="type" type="text" name="user_name" placeholder="your Name *" />
        
                            <input class="type" type="email" name="email" placeholder="your e-mail *" />
        
                            <input class="type" type="text" name="subject" placeholder="subject" />
                        </li>
                        <li class="footer-list-item">
                            <textarea class="coment" placeholder="Your message *"></textarea>
                        </li>
                    </ul>
                    <div class="btn-big">
                        <button type="submit" class="btn">Send Message</button>
                    </div>
            </form>
        </div>
    </footer>
</body>
</html>